tests/e2e/bq/bq-batch-retrieval.py

```diff
@@ -186,7 +186,7 @@ def test_batch_get_historical_features_with_file(client):
     client.ingest(file_fs1, features_1_df, timeout=480)
 
     # Rename column (datetime -> event_timestamp)
-    features_1_df["datetime"] + pd.Timedelta(
+    features_1_df["datetime"] += pd.Timedelta(
```

Comment:
I know you didn't add this buffer, but isnt it counter-intuitive that it needs to be added? Shouldn't the tests be representative of how users will be using the system?